irrtree/cli.py

```diff
@@ -277,6 +277,93 @@ def main():
 
     process(irr_host, afi, db, query_object, search)
 
+  
+ def main_export(*data):
+
+    global debug
+    debug = False
+
+    irr_host = 'rr.ntt.net'
+    irr_port = 43
+    afi = 4
+    search = False
+    sources_list = False
+
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], "h:dp:64s:l:",
```

Comment:
as you `import` the code and don't use it as a standalone CLI utility, I think you don't need `getopt`